hw_11/index.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+error_reporting(E_ALL);
+ini_set('display_errors',1);
+
+/*///////////////////////////////// NORMAL LEVEL ///////////////////////////////////////////////// */
+
+/*
+//Task 1
+$fruits = ["Apple", "Banana", "Orange"];
+echo count($fruits);
+*/
+
+/*
+//Task 2
+$student = [
+    'firstName' => 'Nastya',
+    'lastName' => 'Balakhomskaya',
+    'age' => 25,
+];
+*/
+
+/*
+//Task 3
+$message ='Hello';
+
+echo $message[0] . $message[1] . $message[4];
+*/
+
+/*
+//Task 4
+//$arr = [1, 2, 3, 4, 5];
+$arr = array (1, 2, 3, 4, 5);
+echo '<pre>';
+print_r($arr);
+echo  '</pre>';
+*/
+
+/*
+//Task 5
+$arr = [
+    'ru' => ['голубой', 'красный', 'зеленый'],
+    'en' => ['blue', 'red', 'green'],
+];
+
+echo $arr['ru'][0], $arr['ru'][1], $arr['en'][1];
+*/
+
+/*
+//Task 6
+$arr = ['Hello ', 'world', '!'];
+
+print_r(implode($arr)); //implode - объединение элементов массива
+*/
+
+//Task 7
+/*
+// 1 способ (через if)
+$lang = 'en';
+
+if ($lang === 'ru') {
+    $arr = ['Понедельник', 'Вторник', 'Среда', 'Четверг', 'Пятница', 'Суббота', 'Воскресенье'];
+} elseif ($lang === 'en') {
+    $arr = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+}
+print_r($arr);
+*/
+
+/*
+// 2 способ (через switch-case)
+$lang = 'ru';
+
+switch ($lang) {
+    case 'ru':
+        $arr = ['Понедельник', 'Вторник', 'Среда', 'Четверг', 'Пятница', 'Суббота', 'Воскресенье'];
+        break;
+    case 'en':
+        $arr = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+        break;
+    default:
+       echo 'Выберите не эльфийский язык)))';
+        break;
+}
+print_r($arr);
+*/
+
+/*
+// 3 способ (через match)
+$lang = 'ru';
+
+$dayweek = match ($lang) {
+    'ru' => $arr = ['Понедельник', 'Вторник', 'Среда', 'Четверг', 'Пятница', 'Суббота', 'Воскресенье'],
+    'en' => $arr = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
```

Comment:
match и так вернет результат
```suggestion
    'ru' => ['Понедельник', 'Вторник', 'Среда', 'Четверг', 'Пятница', 'Суббота', 'Воскресенье'],
    'en' => ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
```